### ISSUE TITLE
Add all repositories feature to Community toolbox dropdown

### ISSUE THREAD

**Issue by imajit** [publiclab/community-toolbox#323]
The current dropdown in [Community Toolbo](https://code.publiclab.org/) is useful in filtering issues based on repository but there is no option to reset to all repositories. 
![Screenshot from 2020-01-23 00-11-52](https://user-images.githubusercontent.com/38528640/72924077-4322a280-3d76-11ea-987f-b69a73fd87c1.png)
So along with the repositories it would be great to have an option to view issues of all repositories.
Basically get back to https://code.publiclab.org/#r=all

**Comment by govindgoel** [comment]
@imajit  |https://code.publiclab.org/#r=all| If you navigate to this link you will get all repositories although    it list all the repositories.

**Comment by Rishabh570** [comment]
Yes, there's no "all" option in the dropdown list as of now, the reason being, the contents of the dropdown are set dynamically which are sourced from the DB. It is a good thing to work on though.
`/utils/navDropdown.js` would be the right place to do the modification I guess.

Thanks for creating this issue @imajit :+1:.

**Comment by imajit** [comment]
Hey ! Can I work on this issue ?

**Comment by govindgoel** [comment]
@Rishabh570  @imajit  I fixed the issue can i open a PR for this.

**Comment by Rishabh570** [comment]
@govindgoel Please go ahead!
